setup.py

```diff
@@ -50,6 +50,8 @@ def read_requirements(file_):
             "bugbug-microannotate-generate = scripts.microannotate_generator:main",
             "bugbug-classify-commit = scripts.commit_classifier:main",
             "bugbug-regressor-finder = scripts.regressor_finder:main",
+            "bugbug-generate-sheet = scripts.generate_sheet:main",
+            "bugbug-retrieve-training-metrics = scripts.retrieve_training_metrics:main",
         ]
```

Comment:
Again, https://github.com/mozilla/bugbug/pull/837#issuecomment-517657249 :P
These should no longer be console_scripts, but scripts.

You'll also need to rebase on top of master, as a few more scripts have been added recently.